packages/ui/Routable/Link.js

```diff
@@ -0,0 +1,69 @@
+import kind from '@enact/core/kind';
+import {forProp, forward, handle, preventDefault} from '@enact/core/handle';
+import hoc from '@enact/core/hoc';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import {RouteContext} from './util';
+
+const LinkBase = kind({
+	name: 'Link',
+
+	propTypes: {
+		path: PropTypes.string.isRequired,
+		disabled: PropTypes.bool
+	},
+
+	defaultProps: {
+		disabled: false
+	},
+
+	handlers: {
+		onClick: handle(
+			forProp('disabled', false),
+			forward('onClick'),
+			preventDefault
+		)
+	},
+
+	render: ({path, ...rest}) => {
+		return (
+			<a href={path} {...rest} />
+		);
+	}
+});
+
+const Linkable = hoc({navigate: 'onClick'}, (config, Wrapped) => {
+	const {navigate} = config;
+
+	return kind({
+		name: 'Linkable',
+
+		contextType: RouteContext,
+
+		propTypes: {
+			path: PropTypes.string.isRequired
```

Comment:
For some reason, this reqire specifically, and no other, is firing. Removing `isRequired` here resolves the issue with no additional warnings or errors.

Reply:
Did you see those prop-type warnings in settings? If so, it's because `MenuItem` is wrapped by `Routable` by not all of them have a path (because we haven't build those views out) so it warns.